zerver/views/webhooks/heroku.py

```diff
@@ -0,0 +1,24 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from typing import Any, Callable, Iterable, Optional, Tuple, Text
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.lib.validator import check_dict
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Stream, Client
+
+
+@api_key_only_webhook_view("Heroku")
+@has_request_variables
+def api_heroku_webhook(request, user_profile, client, stream=REQ(),
```

Comment:
Can we default the stream name to be "heroku"?  If you look at other webhooks, you should find examples on how to do this.